docker/redis.persistent.conf

```diff
@@ -13,5 +13,5 @@ aof-use-rdb-preamble yes
 save ""
 
 # Memory management
-maxmemory 512mb
+maxmemory 2gb
```

Comment:
Thats a drastic increase 4 times, probably the docs or some place this require a update?

Reply:
IIRC, we actually use `4gb` in production. It made more sense to raise it up to a more realistic level given the size of the mainnet.